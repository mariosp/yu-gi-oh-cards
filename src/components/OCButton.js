import React from 'react';



const OCButton = ()=>(
   

    <div  id="ocbutton" className="container" onClick={()=> {
        document.getElementById('ocbutton').classList.toggle('change');
        }
    }>
        <div className="bar1"></div>
        <div className="bar2"></div>
        <div className="bar3"></div>
    </div>
    

);

export default OCButton;