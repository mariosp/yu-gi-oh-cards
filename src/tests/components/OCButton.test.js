import React from 'react'
import {shallow} from 'enzyme';
import OCButton from '../../components/OCButton';


test('should render OCButton correctly', ()=> {

    const wrapper = shallow(<OCButton />);
    expect(wrapper).toMatchSnapshot(); 

}); 

