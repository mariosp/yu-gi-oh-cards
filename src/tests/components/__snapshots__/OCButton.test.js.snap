// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render OCButton correctly 1`] = `
<div
  className="container"
  id="ocbutton"
  onClick={[Function]}
>
  <div
    className="bar1"
  />
  <div
    className="bar2"
  />
  <div
    className="bar3"
  />
</div>
`;
